Meadowlark-js/tests/e2e/functions/BulkCreation.ts

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { createSchool } from './DataCreation';
+import { deleteResourceByLocation } from './Resources';
+
+class Resources {
```

Comment:
Absent a compelling reason to be a class, this is better off as simply a type.

Reply:
More generally, note that while in C#-land classes also incorporate the concepts of types and modules, in TypeScript-land, they are all separable.